01_fib/ovlords/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+
+	if n > 0 {
+
+		currentNumber, previousNumber, scratchNumber := 1, 0, 0
+
+		for lines := 1; lines <= n; lines++ {
```

Comment:
Go programs will usually use `i` as a loop variable when it is purely for counting an index.